modules/ROOT/pages/security/ref_terminology.adoc

```diff
@@ -0,0 +1,34 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: ref_terminology_{device-security-service}
+
+[id='{context}_ref_terminology']
+
+= Provided Self Defence Checks
+
+The following are the self defence checks provided by the SDK.
```

Comment:
As discussed i think we should use a tabbed box here and copy over the content from the various platform SDKs.